src/AppBundle/Resources/public/js/contacts.js

```diff
@@ -56,6 +56,41 @@ $(function () {
         });
     }
 
+    function filterCities () {
+        var hideBlock = true;
+
+        $('.social').each(function () {
+            var cityId = $(this).data('city_id');
+            var show = false;
+
+            if (selectedCities.length === 0) {
+                //user did not specify any filters
+                show = true;
+                hideBlock = false;
+            }
+
+            if (selectedCities.includes(cityId)){
+                show = true;
+                hideBlock = false;
+            }
+
+            if (show) {
+                $(this).fadeIn();
+            } else {
+                $(this).fadeOut();
+            }
+        });
+
+        console.log(hideBlock)
+        if (hideBlock){
```

Comment:
@daria-sieroshtan 
It's easier not to store this as variable but unconditionally hide at the beginning and show whenever the reason for show is found